src/bookshelf/rebar.config

```diff
@@ -95,6 +95,8 @@
    "\"(bksw_conf)\":\"(reset_dispatch)\"/\".*\"" %% used in tests
    " - "
    "\"(bksw_wm_sql_object|bksw_wm_object)\":\"(validate_content_checksum|upload|download)\"/\".*\"" %% content_types_accepted-defined callbacks,
+   " - "
+   "\"(bksw_sec)\":\"(test)\"/\".*\"" %% flagged as error, but it's nonexistent?
```

Comment:
do we still need this change? looks like the verify tests are successful here and in https://github.com/chef/chef-server/pull/2097 without this change.

Reply:
I'll pull the commit and see what happens.